pkg/restapi/kms/operation/operations.go

```diff
@@ -438,6 +584,51 @@ func (o *Operation) writeResponse(rw io.Writer, v interface{}) {
 	}
 }
 
+func unmarshalPublicKey(k *publicKey) (*crypto.PublicKey, error) {
```

Comment:
Unwrapped errors

Reply:
thanks, this case is more subtle - will think on this in my next PR